package.json

```diff
@@ -7,16 +7,25 @@
     "prettier-hook": "pretty-quick",
     "prettier-check-hook": "pretty-quick --check",
     "prepublishOnly": "make move-package-json-to-dist",
-    "update-version": "make update-entry"
+    "update-version": "make update-entry",
+    "test": "jest --env=jsdom"
```

Comment:
why do u need jsdom? what does it mean?

Reply:
we need to access the dom during the test because we want to render a component